.github/workflows/continuous-delivery.yml

```diff
@@ -83,6 +83,9 @@ jobs:
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
 
+      - if: ${{ steps.changesets.outputs.hasChangesets == 'true' }}
+        run: yarn run changesets:version
```

Comment:
if you run it like that, it will skip the step with release PR as the version will be bumped. Maybe to just copy the api reference and changelogs you could run `yarn run prepare:docs` in here?